drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -991,8 +991,8 @@ static void usbd_event_transfer_ctrl(nrfx_usbd_evt_t const *const p_event)
 			err_code = nrfx_usbd_ep_status_get(
 				p_event->data.eptransfer.ep, &ep_ctx->buf.len);
 
-			if ((err_code != NRFX_SUCCESS) &&
-			    (err_code != (nrfx_err_t)NRFX_USBD_EP_OK)) {
+			if ((err_code != NRFX_USBD_EP_OK) &&
+			    (err_code != NRFX_USBD_EP_OK)) {
```

Comment:
@galak I guess one of (err_code != NRFX_USBD_EP_OK) is redundant.

Reply:
:), got catch!

Reply:
Looks to be redundant, when I look at what `nrfx_usbd_ep_status_get` does.